docs/articles/developer-info/Coding-Standards.md

```diff
@@ -6,7 +6,7 @@ Notice that these standards are all stylistic. Do not write standards that tell
 
 ## Making Changes
 
-If you do want to make changes, please don't just edit the wiki. Initiate a discussion of what you want to change on the developer list first. If you add an entire new section, you can edit first and then present it to the list for discussion. However, if you are intending to standardize more things than we usually standardize, it's wise to discuss it first to avoid wasting time!
+If you do want to make changes, please initiate a discussion of what you want to change on the developer list first. If you are intending to standardize more things than we usually standardize, it's wise to discuss it first to avoid wasting time!
```

Comment:
@CharliePoole @ChrisMaddock can I get a 👍 from you here to confirm this language still captures the spirit in which it was intended?

Reply:
Yeah, I think is still reflects the original spirit - although I'm not sure I'm totally on-board with a lot of the content or tone in this article! 😅 Stuff like this pre-dates when we started to separate out the different projects a little and possibly all needs reviewing - but that's a bigger task, for another day!

One suggestion for now - the developer list hasn't been used properly for a couple of years. I think the repo's are a better place to discuss things like this these days.

```suggestion
If you do want to make changes, please initiate a discussion of what you want to change on the relevant repository first. If you are intending to standardize more things than we usually standardize, it's wise to discuss it first to avoid wasting time!
```

Reply:
While I agree with you 100%, this PR is just for language changes to ensure the wiki language doesn't hang around.

I'll create another issue for us to discuss separately, as that sounds more like a team-wide change/update in terms of process & communication.

Reply:
I'm with Chris on this one. I recommend you only do the minimum necessary work under developer info because it still reflects our organization of 4 or 5 years back.